### DISCUSSION TITLE
create react native library with JSI templates

### DISCUSSION THREAD

**Discussion by chanphiromsok** [callstack/react-native-builder-bob#451] [Ideas]
Hello to RN dev as we know react native bridge is not fast as JSI in the future it would be Fabric with Turbo and I think react-native-builder-blob should have set up template with optional for JSI template for example cmd npx create-react-native-library --template=JSI or TurboNative Module otherwise use JS bridge as template for my current situation I trouble to config CMake 😅 it is not easy to config it
** Update follow tutorial it hard to config CMakeList
https://blog.notesnook.com/getting-started-react-native-jsi/

**Comment by elichai** [comment]
See also https://reactnative.dev/docs/the-new-architecture/pure-cxx-modules

@atlj It would be great to replace the current template for "Turbo Modules with C++" that goes through Objective-C/Kotlin with one that uses JSI bindings without any "native platform" code except for registering/providing the module
